caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -120,12 +133,62 @@
     position: relative;
     padding: 16px;
     display: block;
-    height: 190px;
   }
 
   .slider-card .card .close-button {
-    margin: -27px 20px 20px;
+    margin: 0 20px 20px;
     position: relative;
     z-index: 2;
   }
+
+
+  .slider-card-wrapper .chevron-left {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    z-index: 100;
+    top: 30%;
+    left: -.5%;
+    background-color: white;
+    width: 30px;
+    height: 30px;
+    border-radius: 80%;
+    box-shadow: 0 10px 15px -3px rgba(0 0 0 / 10%), 0 4px 6px -2px rgba(0 0 0 / 6%);
+  }
+
+  .slider-card-wrapper .chevron-left-svg {
+    background: url("../../icons/chevron-left.svg") no-repeat;
```

Comment:
I'm not sure why we need three different arrow SVGs when a single SVG could be just rotated to match the desired direction. Also, `chevron` is not the right name for left and right arrows. They really should just be called `arrow-left` and `arrow-right`. I know these icons already exist for other blocks but just pointing it out. I'll log a maintenance issue to possibly clena this up.

Reply:
Initially the chevrons/arrows were not symmetrical from where I grabbed them (there was space coded in the path). Then I found another location in which the arrows were "correct", aka without that additional space. That is why we ended up with multiple different ones.